### DISCUSSION TITLE
[6 febbraio 2023] Automi

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/automi-calcolabilita-complessita#4] [Automi]
![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/69389461/3b53397b-6cd0-479d-b762-374367d42eb9)

**Comment by teoang** [comment]


**Reply by Exyss** [reply]
Per quanto riguarda la seconda parte, occhio che il fatto che $\delta^\*(q_0, x^k) = \delta^\*(q_f, x^k)$ non implica che $\delta^\*(q_0, x^k) \in F$ visto che nessuna ipotesi può concludere che $\delta^\*(q_f, x^k) \in F$

**Comment by Exyss** [comment]
Per il primo punto, ripropongo la soluzione presente sui miei appunti:

![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/65043207/e4ee4594-04c7-49b1-af42-129fd90944dc)

----

La dimostrazione richiesta nel secondo punto è presente sul libro di testo

**Reply by dahKidou** [reply]
Ma il primo punto del primo esercizio si può fare anche per induzione sulla lunghezza dell' input?
Io ho fatto una cosa del genere:
![deluyhr](https://github.com/user-attachments/assets/c059d029-eff1-4514-a79f-e93a2c6d7eff)

**Reply by Exyss** [reply]
Sì andrebbe bene lo stesso, l'argomento è analogo tanto.

**Comment by daromacarlo** [comment]
<img width="1244" height="682" alt="Immagine 2026-01-02 172046" src="https://github.com/user-attachments/assets/e5829cbf-d52a-40f0-bba1-e929651a822f" />
<img width="1247" height="712" alt="Immagine 2026-01-02 172101" src="https://github.com/user-attachments/assets/9448e0b3-d831-4816-8562-681568376eb3" />
<img width="1245" height="442" alt="Immagine 2026-01-02 172112" src="https://github.com/user-attachments/assets/91d9e6ea-8f7c-48f5-b342-3e86791a0ffe" />
<img width="983" height="780" alt="Immagine 2026-01-02 175059" src="https://github.com/user-attachments/assets/bcb356ad-6387-4f2d-a5f6-ac79f665c206" />
<img width="984" height="299" alt="Immagine 2026-01-02 175115" src="https://github.com/user-attachments/assets/3fc9f3ea-f861-4737-83c1-e376f7e0baaa" />
<img width="981" height="610" alt="Immagine 2026-01-02 175127" src="https://github.com/user-attachments/assets/9438b0ac-b8b1-4527-827d-93349259b9a9" />

**Reply by PatatineFritte0** [reply]
ma quindi le dimostrazioni delle affermazioni si possono omettere?

**Reply by Exyss** [reply]
È una scelta personale quanto dimostrare e quanto nel dettaglio andare. Chiaramente più dimostri meglio è, però devi fare anche scendere a patti coi limiti di tempo. Nel caso di questa dimostrazione penso sia opportuno omettere le dimostrazioni delle due affermazioni visto che diventerebbe veramente troppo lunga.
